### ISSUE TITLE
[Feature Request] Display chunks from all selected files in the preview

### ISSUE THREAD

**Issue by kolbyjack** [ForkIssues/TrackerWin#786]
Currently, if you select more than one file in the unstaged or staged changes list, only the changes from the first selected files will appear in the preview.  Being able to see and stage lines/hunks from multiple files (like a .cpp and its .h) at once in the preview is the one thing I miss from when I used SourceTree.

**Comment by ondrudav** [comment]
Yeah, this makes reviewing small changes across many files needlessly difficult. The diff view is completely unused when a directory is selected, when it could show changes across all the files it contains.
